.github/workflows/changesets.yml

```diff
@@ -98,3 +83,52 @@ jobs:
           payload: '{"version":"${{ matrix.package.version }}"}'
         env:
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_RELEASE_ANNOUNCEMENT_WEBHOOK_URL }}
+
+  # If a normal release off dev was done, we need to merge into master
+  merge-dev-into-master:
+    needs:
+      - release
+
+    if: ${{ github.ref_name == 'dev' && needs.release.outputs.published == 'true' }}
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: master
+          fetch-depth: 0
+          token: ${{ secrets.NEO4J_TEAM_GRAPHQL_PERSONAL_ACCESS_TOKEN }}
```

Comment:
doesn't this checkout step need to happen on both `merge-dev-into-master` and `merge-dev-into-dev`?

Reply:
This is a good spot, thank you!